components/inputs/docs/styling-native-inputs.md

```diff
@@ -0,0 +1,153 @@
+  import { checkboxStyles } from '@brightspace-ui/core/components/inputs/input-checkbox.js';
+
+  class MyCheckboxElem extends LitElement {
+
+    static get styles() {
+      return checkboxStyles;
+    }
+
+    render() {
+      return html`<input type="checkbox" class="d2l-input-checkbox">`;
+    }
+
+  }
+  customElements.define('d2l-my-checkbox-elem', MyCheckboxElem);
+</script>
+<d2l-my-checkbox-elem></d2l-my-checkbox-elem>
+```
+
+## Radio Inputs
+
+### With Labels
+
+The simplest way to apply radio styles is to use the `d2l-input-radio-label` CSS class on a `<label>` element that wraps the input.
+
+For disabled items, add the `d2l-input-radio-label-disabled` class on the label and the `disabled` attribute on the input itself.
+
+<!-- docs: demo code display:block -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { radioStyles } from '@brightspace-ui/core/components/inputs/input-radio-styles.js';
+
+  class MyRadioElem extends LitElement {
+
+    static get styles() {
+      return radioStyles;
+    }
+
+    render() {
+      return html`
+        <label class="d2l-input-radio-label">
+          <input type="radio" name="myGroup" checked>
+          Option 1 (selected)
+        </label>
+        <label class="d2l-input-radio-label d2l-input-radio-label-disabled">
+          <input type="radio" name="myGroup" disabled>
+          Option 2 (disabled)
+        </label>
+        <label class="d2l-input-radio-label">
+          <input type="radio" name="myGroup">
+          Option 3
+        </label>
+      `;
+    }
+
+  }
+  customElements.define('d2l-my-radio-elem', MyRadioElem);
+</script>
+<d2l-my-radio-elem></d2l-my-radio-elem>
+```
+
+### Individual Radio Inputs
+
+If you'd like to manually link the radio input with a label, or use an ARIA label, place the `d2l-radio-input` CSS class on the input itself to style it. For example:
+
+<!-- docs: demo code properties name:d2l-test-input-radio-solo display:block -->
+```html
+<script type="module">
+  import { html, LitElement } from 'lit';
+  import { radioStyles } from '@brightspace-ui/core/components/inputs/input-radio-styles.js';
+
+  class MyRadioElem extends LitElement {
+
+    static get properties() {
+      return {
+        checked: { type: Boolean },
+        disabled: { type: Boolean },
+        invalid: { type: Boolean }
+      };
+    }
+
+    static get styles() {
+      return radioStyles;
+    }
+
+    render() {
+      const invalid = this.invalid ? 'true' : 'false';
+      return html`
+        <input
+          aria-invalid="${invalid}"
+          aria-label="Option 1"
+          ?checked="${this.checked}"
+          class="d2l-input-radio"
+          ?disabled="${this.disabled}"
+          type="radio">
+      `;
+    }
+
+  }
+  customElements.define('d2l-test-input-radio-solo', MyRadioElem);
+</script>
+<d2l-test-input-radio-solo></d2l-test-input-radio-solo>
+```
+
+### Radio Spacer [d2l-input-radio-spacer]
```

Comment:
I want to delete this component as it's now redundant with `<d2l-input-radio>`'s `supporting` slot.